tests/dag_processing/test_processor.py

```diff
@@ -1035,7 +1035,7 @@ def test_counter_for_last_num_of_db_queries(self):
 
         with create_session() as session:
             with assert_queries_count(
-                expected_count=94,
+                expected_count=154,
```

Comment:
@shahar1 as I asked in this [comment](https://github.com/apache/airflow/pull/40703#issuecomment-2270038205), this change is required because we have now more permissions in dag level to sync:
https://github.com/apache/airflow/blob/27ba034302c1109f721b667326fb14963a768f1d/airflow/providers/fab/auth_manager/security_manager/override.py#L1120-L1123

Reply:
It makes sense to me